pkg/platform/authentication/auth.go

```diff
@@ -49,6 +49,8 @@ type Configurable interface {
 	Close() error
 }
 
+const ApiTokenConfigKey = "apiToken"
+
```

Comment:
Please drop any code change regarding saving the authenticated state. This is already implemented, please use:

http://github.com/ActiveState/cli/blob/950bf1083327954d4a1e807c14de37fd26d771f7/pkg/platform/authentication/auth.go#L293-L293

For example:

http://github.com/ActiveState/cli/blob/950bf1083327954d4a1e807c14de37fd26d771f7/pkg/platform/authentication/auth.go#L194-L194

Reply:
Good catch.